crates/runtime/src/accelerated_table/refresh_task/streaming_append.rs

```diff
@@ -86,12 +86,12 @@ impl RefreshTask {
         Ok(())
     }
 
-    fn get_append_stream(
+    async fn get_append_stream(
         &self,
     ) -> impl Stream<Item = crate::accelerated_table::Result<(Option<SystemTime>, DataUpdate)>>
     {
-        let ctx = self.refresh_df_context();
-        let federated = Arc::clone(&self.federated);
+        let federated = self.federated.table_provider().await;
```

Comment:
Do streaming sources perpetually show up as Deferred? Or will it eventually resolve?

Reply:
I'm not sure I understand the question. The `self.federated.table_provider()` will eventually resolve once the federated source is available (if it wasn't already).